vcr/pe/presentation_definition_test.go

```diff
@@ -710,7 +710,7 @@ func TestPresentationDefinition_ResolveConstraintsFields(t *testing.T) {
 			"any_credential": jwtCredential,
 		}
 
-		fieldValues := definition.ResolveConstraintsFields(credentialMap)
+		fieldValues, _ := definition.ResolveConstraintsFields(credentialMap)
```

Comment:
this one probably needs an `assert.NoError` check to make sure`fieldValues` is not empty due to an error